docs/reference/ois/next/specification.md

```diff
@@ -594,8 +589,16 @@ corresponding operation parameter.-->
 
 ### 5.9. `preProcessingSpecifications` \*
 
-(Optional) Defines the preprocessing code that can be used to modify the
-endpoint parameter before making the API request defined by an Airnode endpoint.
+::: deprecation warning
```

Comment:
We are not looking to remove it just yet, but we want encourage users to migrate so I wanted to give some encouragement message.

Reply:
This isn't rendering because `::: deprecation` isn't a container type. I think you want:
```suggestion
::: warning Deprecation
```

Reply:
Thanks for noticing.